File: CPP/1409A.cpp
#include <iostream>

using namespace std;

int main()
{
  int k, a, b;

  for(cin >> k; k--;)
  {
    cin >> a >> b;
    cout << abs((a - b) / 10) + ((a - b) % 10 ? 1 : 0) << endl;
  }
}

File: Haskell/1409A.hs
import Control.Monad

main :: IO ()
main = getLine >>= (\n -> putStr . unlines . map ((show . (\[x, y] -> f x y)) . map read . words) =<< replicateM n getLine ) . read
  where f a b = (\(x, y) -> x + fromEnum (nz y)) $ abs (a - b) `divMod` 10
        nz = (/= 0)